project_tests.sh

```diff
@@ -3,7 +3,12 @@ set -e
 
 rm -f build/*.xml
 bin/console cache:clear --env=test --no-warmup
-bin/console security:check
+if bin/console security:check; then
```

Comment:
Is this just making it clearer that the message is from the security check? Sounded like it was trying to hide the message.